styleguide/source/_patterns/03-templates/04-topic/_topic.scss

```diff
@@ -38,6 +38,13 @@
       grid-area: 1 / 1 / 4 / 1;
       padding-left: 14px;
     }
+
+    .grid-region_content.topic_topic-articles > div {
```

Comment:
Ugh... so... Drupal is rendering these as `<article>` and the style guide is rendering them as `<div>`. I'm not sure if fixing one or the other is in scope here, but we need to at least change this so it's not calling divs. Any reason it couldn't be `> .topic_article-preview`?